lib/request-proxy/index.js

```diff
@@ -197,18 +219,22 @@ proto.handleRequest = function handleRequest(head, body, cb) {
     httpRequest.headers = headers;
     httpRequest.method = method;
     httpRequest.httpVersion = httpVersion;
-
     httpRequest.end(body);
 
-    var httpResponse = hammock.Response(onResponse);
+    if (this._shouldApplyBackpressure('ingress', function onApply(err) {
+        cb(err);
+    })) {
+        return;
+    }
 
-    ringpop.logger.trace('handleRequest emitting request', {
-        url: url
-    });
+    this._incrementInflightRequests();
```

Comment:
Am I mistaken or does doing this accounting here mean that we count requests both coming and going?  I'm not saying that's wrong, but wouldn't it have the interesting effect that if requests are evenly distributed across the cluster, our in-flight limit in practice is 1/2 of the nominal value?  Is this intentional?